cpp/src/io/avro/avro.cpp

```diff
@@ -67,8 +67,10 @@ std::string container::get_encoded()
 bool container::parse(file_metadata* md, size_t max_num_rows, size_t first_row)
 {
   constexpr uint32_t avro_magic = (('O' << 0) | ('b' << 8) | ('j' << 16) | (0x01 << 24));
-  uint32_t sig4, max_block_size;
+  uint32_t sig4, max_block_size, num_rows, row_offset;
```

Comment:
Please initialize to zero if there's no way to declare at the point of initialization.